ReactWindows/js/Components/Picker/PickerAndroid.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @providesModule PickerAndroid
+ */
+'use strict';
+
+var React = require('React');
+
+var PickerAndroid = React.createClass({
+  render: function() {
+  },
+});
+
+module.exports = PickerAndroid;
```

Comment:
Is there a better way to deal with `Picker.js` requires than having these two dumb files?
`var PickerIOS = require('PickerIOS');`
`var PickerAndroid = require('PickerAndroid');`